config/initializers/1_settings.rb

```diff
@@ -125,6 +123,20 @@ def verify_constant(modul, current, default)
 Settings.gitlab['repository_downloads_path'] = File.absolute_path(Settings.gitlab['repository_downloads_path'] || 'tmp/repositories', Rails.root)
 
 #
+# Outgoing emails
+#
+Settings['outgoing_emails'] ||= Settingslogic.new({})
+Settings.outgoing_emails['enabled']           ||= Settings.gitlab['email_enabled'] unless Settings.gitlab['email_enabled'].nil?  # for backward compatibility
+Settings.outgoing_emails['enabled']           ||= true if Settings.outgoing_emails['enabled'].nil?
```

Comment:
Yeah but we decided on a styleguide. I'm asking you to follow it. If not i'm forced to close this PR. I'm sorry!